### PR TITLE
Updates to track prediction (for demo_colmap.py)

### PR DESCRIPTION
**Pull request by tommybotch**
Currently, predict_tracks_in_chunks doesn't detach tensors from GPU on each loop ([see here](https://github.com/facebookresearch/vggt/blob/44b3afbd1869d8bde4894dd8ea1e293112dd5eba/vggt/dependency/vggsfm_utils.py#L286-L295)). 

I think this is partially causing related memory issues with `demo_colmap.py` as reported in https://github.com/facebookresearch/vggt/issues/238. 

In addition, memory load seems to be most affected by max_points_num (dropping this number increases chunking and lowers memory load).

### CODE REVIEW

**Comment by 1e100** [comment]
FWIW, this doesn't seem to solve the OOMs for me. GPU: A6000 (Ampere). Is there a reason to `.detach()` when the tensors were already created under the `no_grad()` context manager and do not require gradients?
